lib/performance/mark.js

```diff
@@ -7,6 +7,8 @@
 // Imports
 var global = require('../global');
 var perfNow = require('./now.js');
+var marks = require('./entries').marks;
```

Comment:
This property is currently not read by any other module, correct?

Reply:
Looking through the code, that does appear to be the case. I think once the timeline started being used in multiple modules, it made sense to me to also house the marks object separately in case of the same needs, but I'm not even importing it as part of the test modules. Also, I'm not even sure that `marks` is even needed at all anymore, because we never previously exposed a method to `getEntries`. It looks like any marks that were set with the polyfill method were simply inaccessible. The new getEntries* family of methods all use the timeline object, which we're also pushing mark records into within the mark module, so that object may be obsolete at this point.